.github/workflows/Release.yaml

```diff
@@ -108,8 +108,9 @@ jobs:
             New-AzStorageContainer -Name 'public' -Context $storageContext -Permission 'Container' -ErrorAction Ignore | Out-Null
           
             Compress-Archive -path $path -DestinationPath "$($path).zip"
+            $cacheDurationSeconds = 3600 # 1 hour
             Set-AzStorageBlobContent -File "$($path).zip" -Context $storageContext -Container 'public' -Blob "$version.zip" -Force | Out-Null
-            Set-AzStorageBlobContent -File "$($path).zip" -Context $storageContext -Container 'public' -Blob "latest.zip" -Force | Out-Null
+            Set-AzStorageBlobContent -File "$($path).zip" -Context $storageContext -Container 'public' -Blob "latest.zip" -Properties @{"CacheControl" = "max-age=$cacheDurationSeconds"} -Force | Out-Null

```

Comment:
I think we should do no-cache on both.
Reason being that we would need to always remember to release a new BcContainerHelper 1 hour before we release AL-Go, because you could have the old version cached for an hour.